etc/nebula-graphd.conf.default

```diff
@@ -1,24 +1,24 @@
 ########## basics ##########
-# Whether to run as a daemon process
+# Enabling/Disabling daemon process
```

Comment:
To be honest, i prefer the original expression.

Reply:
but i think the modified are more clear.